tox.ini

```diff
@@ -29,6 +29,7 @@ commands =
 deps =
     2.2: Django==2.2.*
     3.0: Django==3.0.*
+    3.1: Django==3.1.*
```

Comment:
Merely adding this line is not enough to let tox run the tests with 3.1, you should also update `envlist`:

```
    py36-{2.2,3.0,3.1,master},
    py37-{2.2,3.0,3.1,master},
    py38-{3.0,3.1,master},
    py39-{3.0,3.1,master},
```